src/Sylius/Behat/Page/Shop/Product/ShowPage.php

```diff
@@ -226,9 +227,32 @@ public function isMainImageDisplayed()
     /**
      * {@inheritdoc}
      */
+    public function hasAssociation($productAssociationName)
+    {
+        return null !== $this->getElement('association', ['%association-name%' => $productAssociationName]);
```

Comment:
`$this->getElement` will throw exception when element will not be found. We should try-catch to return boolean instead of exception.